cmd/res/profiles/generic.yaml

```
@@ -7,25 +7,57 @@ description: "Example of Device-Uart"
 
 deviceResources:
 -
-    name: "Read_UART"
+    name: "Read_String"
     isHidden: false
-    description: "used to read from the UART device"
+    description: "Read string from the UART device"
     attributes: { type: "generic", maxbytes: 160, timeout: 1}
     properties:
         valueType: "String"
         readWrite: "R"
```

The incoming command determines if it is a Read or a Write so we don't need separate resources for Read and Write

```suggestion
        readWrite: "RW"
```

---

Addressed.